akamai-edgeworkers

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env node
+
 // stub file to kick off compiled app
-require('./bin/cli-main.js');
+// top level CLI needs the file to be started with akamai
+require('./bin/src/edgeworkers/ewcommand.js');
```

Comment:
I wonder if it would be more clear if instead of ewcommand.js and ekvcommand.js you named these files ew-cli-main.js and ekv-cli-main.js?  At least if you upgraded from a previous version and were looking at the source it would be clear what these files are.